### ISSUE TITLE
Add missing contentDescription for ImageButtons and ImageViews

### ISSUE THREAD

**Issue by RajaVamsi11** [openMF/mobile-wallet#401]
**Summary:**

The ContentDescription for ImageButtons and ImageViews is missing in the project. This is the Accessibility feature of Android. Non-textual widgets like ImageViews and ImageButtons should use the content description attribute to specify a textual description of the widget such that screen readers and other accessibility tools can adequately describe the user interface. 
My idea is to add the content description to the ImageButtons and ImageViews to fix the Android (Image without content description) **Accessibility lint warnings**.  For more info you can refer [here](https://developer.android.com/guide/topics/ui/accessibility/apps#label-elements).

**Comment by RajaVamsi11** [comment]
@naman14 @luckyman20 I am working on this issue.

**Comment by therajanmaurya** [comment]
Fixed in compose implementation
